src/main/java/nextstep/courses/domain/SessionImage.java

```diff
@@ -0,0 +1,58 @@
+package nextstep.courses.domain;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class SessionImage {
+    public static final int MB = 1024 * 1024;
+    public static final List<String> IMG_FILE_TYPE = List.of("gif", "jpg", "jpeg", "png", "svg");
+    private long fileSize;
+    private String fileType;
+    private ImageSize imageSize;
+
+
+    public SessionImage() {
+    }
+
+    public SessionImage(File file) {
+        this.fileSize = validateFileSize(file);
+        this.fileType = validateFileType(file);
+        this.imageSize = validateImageSize(file);
+    }
+
+    public SessionImage(long fileSize, String fileType, ImageSize imageSize) {
+        this.fileSize = fileSize;
+        this.fileType = fileType;
+        this.imageSize = imageSize;
+    }
+
+    private Long validateFileSize(File file) {
+        if (file.getTotalSpace() > 1*MB) {
+            throw new IllegalArgumentException();
```

Comment:
```suggestion
        if (file.getTotalSpace() > 1 * MB) {
            throw new IllegalArgumentException();
```
왜 예외가 발생되었는지 구체적인 예외 사항을 전달해주시면 좋을 것 같아요 :)